x-pack/plugins/infra/server/lib/sources/sources.test.ts

```diff
@@ -50,42 +50,9 @@ describe('the InfraSources lib', () => {
       });
     });
 
-    test('adds missing attributes from the static configuration to a source configuration', async () => {
```

Comment:
this is testing for kibana config values in `sources` field, which is no longer supported as part of source configuration.